packages/api-client/src/components/DataTable/DataTableInput.vue

```diff
@@ -11,7 +11,10 @@ const props = withDefaults(
     id?: string
     type?: string
     /** Class for the wrapping cell because attrs is bound to the input */
-    containerClass?: string
+    containerClass?:
```

Comment:
couldn't find the class for this in vue 🕵🏾

Reply:
Looks like Vue uses `any`? Maybe we should just make our own utility type.

![Cursor-2024-11-26-14-16-24@2x](https://github.com/user-attachments/assets/3a19e9b6-e38d-45db-9e3f-76877a128029)

Reply:
lets do it